src/main/java/frc/robot/subsystems/BBalanceSubsystem.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.hardware.MotorController;
+import frc.robot.hardware.MotorController.MotorConfig;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkMaxPIDController;
+
+public class BBalanceSubsystem extends SubsystemBase { // TODO: Find a permanent name for buddy balance subsystem - BBalance is ambiguous
+  private SparkMaxPIDController BB_PIDController;
+  private CANSparkMax rightBBalanceMotor;
+  private boolean toggle = false;
+
+  public BBalanceSubsystem() {
+    rightBBalanceMotor = MotorController.constructMotor(MotorConfig.BBalanceRightShaftRotate);
+    BB_PIDController = rightBBalanceMotor.getPIDController();
+    BB_PIDController.setP(1); // placeholder PID values
+    BB_PIDController.setI(0);
+    BB_PIDController.setD(0);
+    BB_PIDController.setFF(0);
```

Comment:
You don't need to set these to 0, they are 0 by default

PID values should be constants